portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/dataset/DataSetViewActions.testcase

```diff
@@ -2975,7 +2975,7 @@ definition {
 
 		task ("Then the text “No actions were created. Start creating one action to interact with your data.” is displayed above the Create Item Action button") {
 			AssertElementPresent(
-				key_textName = "No actions were created.",
+				key_textName = "No actions were created.Start creating one action to interact with your data.",
```

Comment:
Is there a missing space in `created.Start`?

Reply:
The ` textContent` that returns the selected node does not have spaces.

![Screenshot from 2024-03-18 14-38-10](https://github.com/liferay-frontend/liferay-portal/assets/132653342/0ef1f362-4b5e-4b93-8e53-a96d2e584489)

Reply:
Another option could be changing the [locator](https://github.com/liferay-frontend/liferay-portal/blob/master/portal-web/test/functional/com/liferay/portalweb/paths/portlet/frontenddataset/DataSet.path#L451), so we only retrieve the title, only "No actions were created". 

This test is the only one that uses it.

Reply:
:man_facepalming: 

I guess it's fine like this. We don't want to spend much time on Poshi.